src/test/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollectionTest.java

```diff
@@ -110,6 +112,62 @@ public void testIntervalMergingRuleAdjacentNoMerge(IntervalArgumentCollection ia
         Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()).get(1), new SimpleInterval("1", 101, 200));
     }
 
+
+    @Test(dataProvider = "optionalOrNot")
+    public void testIntervalMergingRuleOverlappingNoMerge(IntervalArgumentCollection iac){
+        iac.addToIntervalStrings("1:1-100");
+        iac.addToIntervalStrings("1:51-200");
+        iac.intervalMergingRule = IntervalMergingRule.NONE;
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()).size(), 2);
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()).get(0), new SimpleInterval("1", 1, 100));
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()).get(1), new SimpleInterval("1", 51, 200));
+    }
+
+    @Test(dataProvider = "optionalOrNot")
+    public void testIntervalMergingRuleOverlappingNoMergeWithPadding(IntervalArgumentCollection iac){
```

Comment:
Add a separate test case with intervals that are non-overlapping, but do overlap when you add the padding.

Reply:
done